test/e2e/handler/nnce_conditions_test.go

```diff
@@ -37,7 +37,7 @@ func invalidConfig(bridgeName string) nmstate.State {
 `, bridgeName))
 }
 
-var _ = Describe("[rfe_id:3503][crit:medium][vendor:cnv-qe@redhat.com][level:component]EnactmentCondition", func() {
+var _ = Describe("EnactmentCondition", func() {
```

Comment:
I saw that the new specReport now starts with Texts[0] instead of Texts[1]
for the old reporters 
https://github.com/nmstate/kubernetes-nmstate/pull/1002/commits/3a5358141b37477b132f2751393339653827e0c9#diff-6b11a0d0638c5aea0a09de9f7fa737dbfee413cf63d52940c706aa7c4b486dbdR136

worth to check that the whole string you expect appears please
(start with `EnactmentCondition`)

i don't refer to the removed `rfe_id`, just to what the junit starts with now